organization/financial-model.md

```diff
@@ -12,26 +12,28 @@ The Foundation for Public Code is a member-owned association. Its mission is to
 
 This means we are primarily member-funded, with additional support from grants, subsidies and philanthropic donations.
 
-Our funding is used for codebase stewardship and to support the public code ecosystem. 
+Our funding is used for codebase stewardship and to support the public code ecosystem.
 
 Examples of things we pay for are: codebase stewards' and support staff salaries, travel to work with members, hosting and attending conferences and events, office expenses, and associated materials like the Standard for Public Code.
 
 ## Membership dues: facts
 
-To achieve our mission, our operations prioritize long term sustainability for the benefit of our members. This means all core operations should be covered by membership dues. 
+To achieve our mission, our operations prioritize long term sustainability for the benefit of our members. This means all core operations should be covered by membership dues.
 
 Membership dues:
+
 * can vary between members
-* determine voting rights in the general assembly 
+* determine voting rights in the general assembly
 * can be paid in money, in kind or a combination of both
 * can change from year to year
 * are paid yearly
 
-More information can be found in the [governance model](governance-model.md) and association bylaws.
+More information can be found in the [governance model](governance-model.md) and [association bylaws](association-bylaws-Dutch.md).
```

Comment:
This file `association-bylaws-Dutch.md` does not exist, remove link for now.

When it does the link can be added. Otherwise pull requests are dependent on each other.

Also file should be named with the convention `.nl.md`, so `association-bylaws.nl.md` and without capitals in the file name.